include/openthread/border_router.h

```diff
@@ -117,6 +117,23 @@ otError otBorderRoutingGetOmrPrefix(otInstance *aInstance, otIp6Prefix *aPrefix)
  */
 otError otBorderRoutingGetOnLinkPrefix(otInstance *aInstance, otIp6Prefix *aPrefix);
 
+/**
+ * This function returns the local NAT64 prefix.
+ *
+ * This prefix might not be advertised in the Thread network.
+ *
+ * This function is only available when `OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE`
+ * is enabled.
+ *
+ * @param[in]   aInstance   A pointer to an OpenThread instance.
+ * @param[out]  aPrefix     A pointer to where the prefix will be output to.
+ *
+ * @retval  OT_ERROR_INVALID_STATE  The Border Routing Manager is not initialized yet.
+ * @retval  OT_ERROR_NONE           Successfully retrieved the nat64 prefix.
```

Comment:
nit: for consistency
```suggestion
 * @retval  OT_ERROR_NONE           Successfully retrieved the NAT64 prefix.
```

Reply:
Updated. Thanks!